src/library/scala/collection/IterableOnce.scala

```diff
@@ -1056,10 +1059,13 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
     *
     *  @example  `List(1, 2, 3).mkString("(", "; ", ")") = "(1; 2; 3)"`
     */
-  def mkString(start: String, sep: String, end: String): String =
-    addString(new StringBuilder(), start, sep, end).toString
+  @inline final def mkString(start: String, sep: String, end: String): String =
```

Comment:
Do we want to inline this much code unconditionally? Usually only simple forwarders are marked `inline` (unless they are private methods that are explicitly designed to be inlined)

Reply:
sg, I'll remove `inline` from this and the 3 arg mkString def in StringOps